packages/bull-vault/src/BullStrategy.sol

```diff
@@ -110,8 +111,10 @@ contract BullStrategy is ERC20, LeverageBull {
      * @param _receiver receiver address
      */
     function farm(address _asset, address _receiver) external onlyOwner {
+        require(!IController(powerTokenController).isShutDown(), "BS7");
```

Comment:
I think technically these should only be in the `if (_asset == address(0)) {` section as we only want to restrict farming for ETH in that case.  For example, there could be euler tokens in the contract.  But I don't think it matters too much either way.